packages/airnode-node/src/evm/wallet.ts

```diff
@@ -37,6 +37,10 @@ export function getAirnodeWallet(config: Config): ethers.Wallet {
   return ethers.Wallet.fromMnemonic(mnemonic);
 }
 
+export function getAirnodeWalletWithPrivateKey(privateKey: string): ethers.Wallet {
```

Comment:
I would name it `getAirnodeWalletFromPrivateKey`. Calling it "with private key" suggests that it returns a wallet and a private key. At least it sounds like that to me, it may be just my bad English :smile: